sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/elasticsearch/LargeProductionCluster/es-services.yaml

```diff
@@ -163,9 +164,9 @@ spec:
             add:
 # IPC_LOCK capability is enabled to allow Elasticsearch to lock the heap in memory so it will not be swapped. 
               - IPC_LOCK
+# SYS_RESOURCE capability is set to control and override various resource limits.
```

Comment:
this comment doesn't really explain what's going on. Why do you need SYS_RESOURCE in order to run ES?